draft-packages/table/KaizenDraft/Table/styles.scss

```diff
@@ -209,7 +209,7 @@ $row-height: 60px;
   composes: anchorReset;
 
   .defaultSpacing & {
-    padding: calc(#{$kz-var-spacing-md} * 0.5 #{$kz-var-spacing-md});
```

Comment:
Was this the typo? The second token `$kz-var-spacing-md` should have been outside the `calc()`?